python/hail/expr/expression.py

```diff
@@ -681,7 +736,7 @@ def collect(self):
         """
         uid = Env._get_uid()
         t = self._to_table(uid)
-        return t.aggregate(hl.agg.collect(t[uid]))
+        return [r[uid] for r in t.collect()]
```

Comment:
Note to self: I had an example from the tutorial where this change made a HUGE difference.  Check again after this lit stuff settles down.

Reply:
we can actually optimize it further by selecting just 'uid' frmo the table. I bet the slow thing is collecting Loci, which go through java again.

Reply:
I'll do this change here.